.travis.yml

```diff
@@ -9,7 +9,8 @@ cache:
   bundler: true
   directories:
     - $HOME/tools
-    - "$TRAVIS_BUILD_DIR/src/$COMPONENT/vendor/bundle"
+    - src/supermarket/vendor/bundle
+    - src/supermarket/engines/fieri/vendor/bundle
```

Comment:
Opting to be explicit about caching the bundled gems in the two Ruby code bases.

Reply:
Coming back to this, I think we should add `$TRAVIS_BUILD_DIR` back in if the goal is explicitness.